src/config/config.ts

```diff
@@ -50,48 +41,25 @@ export class RaftConfig {
     return this._subgraphEndpoint;
   }
 
-  static getTokenAddress<T extends Token>(token: T): TokenAddressType[T] {
-    switch (token) {
-      case 'stETH':
-        return this.networkConfig.stEth as TokenAddressType[T];
-
-      case 'wstETH':
-        return this.networkConfig.wstEth as TokenAddressType[T];
-
-      case 'R':
-        return this.networkConfig.r as TokenAddressType[T];
-
-      default:
-        return null as TokenAddressType[T];
+  static getTokenAddress(token: Token): string {
+    try {
+      return this.networkConfig.tokenTickerToTokenConfigMap[token].address;
+    } catch (error) {
+      throw new Error(`Failed to fetch ${token} address!`);
     }
   }
 
   static getTokenTicker(address: string): Token | null {
-    switch (address.toLowerCase()) {
-      case ZeroAddress:
-        return 'ETH';
-
-      case this.networkConfig.stEth.toLowerCase():
-        return 'stETH';
+    const tokenTicker = TOKENS.find(ticker => {
+      if (this.networkConfig.tokenTickerToTokenConfigMap[ticker].address === address) {
+        return true;
+      }
+    });
```

Comment:
The condition is enough:
```suggestion
    const tokenTicker = TOKENS.find(ticker =>
        this.networkConfig.tokenTickerToTokenConfigMap[ticker].address === address
    });
```

and I'd keep `toLowerCase` since someone can pass an address that has a different casing.